php/src/vaas/Vaas.php

```diff
@@ -190,15 +183,12 @@ public function ForFile(string $path, $upload = true, string $uuid = null): Vaas
      * @throws GuzzleException
      * @throws UploadFailedException
      */
-    public function ForStream(Stream $stream, string $uuid = null): VaasVerdict
+    public function ForStream(ReadableStream $stream, int $size = 0, string $uuid = null): VaasVerdict
     {
-        if ($uuid == null) {
-            $uuid = UuidV4::getFactory()->uuid4()->toString();
-        }
-
-        $verdictResponse = $this->_verdictResponseForStream(
-            $uuid
-        );
+        $this->_logger->debug("uuid: ".var_export($uuid, true));
+        $uuid = $uuid !== null ? $uuid : UuidV4::getFactory()->uuid4()->toString();
+        $this->_logger->debug("uuid: ".var_export($uuid, true));
```

Comment:
Double debug print looks like this is a leftover from debugging. A single print in this line is enough debug logging IMHO.

Reply:
Both where debugging stuff before I found out, that in VSCode you just have to start the debugger first, then you will basically debug anything you run in php like starting the test from the testexplorer but also from console. That made my life so much easier.